packages/prisma/seed.ts

```diff
@@ -181,6 +181,19 @@ async function createPlatformAndSetupUser({
         subscriptionId: "sub_123",
       },
     });
+
+    await prisma.platformOAuthClient.create({
+      data: {
+        name: "Acme",
+        redirectUris: ["http://localhost:4321"],
+        permissions: 1023,
+        areEmailsEnabled: true,
+        organizationId: team.id,
+        id: "clxyyy21o0003sbk7yw5z6tzg",
+        secret:
+          "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiQWNtZSAiLCJwZXJtaXNzaW9ucyI6MTAyMywicmVkaXJlY3RVcmlzIjpbImh0dHA6Ly9sb2NhbGhvc3Q6NDMyMSJdLCJib29raW5nUmVkaXJlY3RVcmkiOiIiLCJib29raW5nQ2FuY2VsUmVkaXJlY3RVcmkiOiIiLCJib29raW5nUmVzY2hlZHVsZVJlZGlyZWN0VXJpIjoiIiwiYXJlRW1haWxzRW5hYmxlZCI6dHJ1ZSwiaWF0IjoxNzE5NTk1ODA4fQ.L5_jSS14fcKLCD_9_DAOgtGd6lUSZlU5CEpCPaPt41I",
```

Comment:
Can we confirm if this secret isn't used anywhere @Ryukemeister

Reply:
@hariombalhara yeah I had this generated for my local environment so it's not from prod.